artemis-distribution/pom.xml

```diff
@@ -39,6 +39,11 @@
      <!-- ActiveMQ Artemis artifacts -->
       <dependency>
          <groupId>org.apache.activemq</groupId>
+         <artifactId>artemis-jms-client-all</artifactId>
```

Comment:
You beat me to it, have a version myself locally just hadn’t had time to commit it.

What about inc the core client all jar also?

Reply:
My thought here was to keep it really simple so instead of having a JMS client and a core client having just the JMS client which (by design) includes the core client.  The JMS client is just around 300KB larger.  In a future release I think we'll get ride of the core and JMS "all" clients and just have a single artemis-client-all which will actually just be the JMS "all" client but renamed to be less confusing.

Reply:
That’s a good point / idea. How close are we to release/tag is it would maybe doing that now? I can try do it tonight.

Reply:
Would = worth

Reply:
Btw +1 (with or without suggested change by you) happy if this is merged.

Reply:
I don't think we could remove Maven modules in a minor release (e.g. 2.4.0).  However, we could do it in a major release (e.g. 3.0.0).  I'd like to tag and get the release out for a vote today.

Reply:
Cool, as noted I’m happy for this to merge